### DISCUSSION TITLE
datetime format

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#2469] [Q&A] [answered]
Hi team, 
We are updating Functions Service page and we would like to know what's the recommended datetime format. 
We've talked to Console team and they said the internationalization team recommends using [i18next](https://www.i18next.com/translation-function/formatting#datetime). Is this the right path? 

Thanks.

Sang

**Comment by serifluous** [comment, marked as answer]
Hey Sang! You can reference the internal product docs for what's currently recommended by the Global team: https://internal-product-docs.twilio.com/docs/internationalization/overview/tooling. I'm going to make an update to our docs site, so the internal product docs are easier to reference from our content, so thanks for asking!

For datetime format, take a look at our content product style guide: https://paste.twilio.design/foundations/content/product-style-guide#dates. Content designers are currently out of office right now, but I can tag them in later on if you need more specific guidance.
